CRM/Core/Payment/PayPalProIPN.php

```diff
@@ -648,7 +643,7 @@ public function handlePaymentExpress() {
     $contribution->loadRelatedObjects($input, $ids);
```

Comment:
@seamuslee001 I'll remove this in a follow up - but you can see that with this merged it's clear the dreaded loadRelatedObjects ^^ is no longer needed here